### PR TITLE
bs4 home applications review

### PR DESCRIPTION
**Pull request by charlienparker**
# PR Checklist

Please check if your PR fulfills the following requirements
- [ ] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit)
- [ ] Tests for the changes have been added (for bugfixes/features), they use let helpers and before blocks
- [ ] For all UI changes, there is cucumber coverage
- [ ] Any endpoint touched in the PR has an appropriate Pundit policy. For open endpoints, reasoning is documented in PR and code
- [ ] Any endpoint modified in the PR only responds to the expected MIME types.
- [ ] For all scripts or rake tasks, how to run it is documented on both the PR and in the code
- [ ] There are no inline styles added
- [ ] There are no inline javascript added
- [ ] There is no hard coded text added/updated in helpers/views/Javascript. New/updated translation strings do not include markup/styles, unless there is supporting documentation
- [ ] Code does not use .html_safe
- [ ] All images added/updated have alt text
- [ ] Doesn’t bypass rubocop rules in any way

# PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature (requires Feature flag)
- [ ] Data fix or migration (inert code, no impact until run)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Dependency updates (e.g., add a new gem or update to a version)

# What is the ticket # detailing the issue?

Ticket: 

# A brief description of the changes

Current behavior:

New behavior:

# Feature Flag

For all new feature development, a feature flag is required to control the exposure of the feature to our end users. A feature flag needs a corresponding environment variable to initialize the state of the flag. Please share the name of the environment variable below that would enable/disable the feature and which client(s) it applies to.

Variable name:

- [ ] DC
- [ ] ME

# Additional Context
Include any additional context that may be relevant to the peer review process.

# AppScan CodeSweep Failure
In the event of a failed check on the AppScan CodeSweep step of our GitHub Actions workflow, please review the False Positive protocol outlined here: appscan_codesweep/CODESWEEP_FALSE_POSITIVES_README.MD

Add all required notes to this section if the failure is a suspected false positive.

### CODE REVIEW

File: components/financial_assistance/app/views/financial_assistance/applications/review.html.erb [hunk 1]
```diff
@@ -1,344 +1,450 @@
...
-                      <% else %>
-                        <%= number_to_currency(embedded_document.amount.to_s) %>
-                      <% end %>
-                    </b>
+                      <% amount = number_to_currency(embedded_document.amount.to_s) %>
+                      <%= embedded_document.is_a?(FinancialAssistance::Deduction) ? "(#{amount})" : amount %>
                     </td>
                     <td>
-                      <%= start_to_end_dates(embedded_document) %>
+                      <div class="d-flex justify-content-between">
```
Comment (scaustin34) [inline components/financial_assistance/app/views/financial_assistance/applications/review.html.erb:50]: nitpick- here and on ln26 the extra div is likely unnecessary. Any classes can just be applied to the parent

File: components/financial_assistance/app/views/financial_assistance/applications/review.html.erb [hunk 2]
```diff
@@ -1,344 +1,450 @@
...
-                  </div>
-                <% end %>
-              <% end %>
-              <%= render partial: 'financial_assistance/shared/modal_support_text_household', locals: {key: "net_annual_income"} %>
-            </div>
-          <% end %>
+    <%# Applicants summary %>
+    <% @applicants.each do |applicant| %>
+      <section>
+        <div class="mb-4">
```
Comment (scaustin34) [inline components/financial_assistance/app/views/financial_assistance/applications/review.html.erb:81]: same as above, no reason to add extra nesting but not incorrect.

File: components/financial_assistance/app/views/financial_assistance/applications/review.html.erb [hunk 3]
```diff
@@ -1,344 +1,450 @@
...
 
-          <div class="row row-form-wrapper ptb no-buffer row-height form-bottom-border">
-            <table class="table mb0">
+        <%= render partial: 'financial_assistance/applications/review_and_submit_subsection', locals: {subsection: family_relationships_hash} %>
+      </section>
+    <% end %>
+    <section><%= render partial: 'financial_assistance/applications/review_and_submit_subsection', locals: {subsection: preferences_hash} %></section>
+    <section><%= render partial: 'financial_assistance/applications/review_and_submit_subsection', locals: {subsection: household_hash} %></section>
+
+    <%= link_to l10n("homepage.applications.detail.back"), applications_path, class: "btn btn-primary mt-2" %>
```
Comment (scaustin34) [inline components/financial_assistance/app/views/financial_assistance/applications/review.html.erb:103]: might need to surround with the `h()` wrap here, as good practice

**Review by kristinmerbach — APPROVED** [review]
LGTM

**Review by ATBull81 — APPROVED** [review]

**Comment by kristinmerbach** [comment]
@charlienparker please rebase from clean-bootstrap-4-upgrade, then this can be merged in

**Comment by charlienparker** [comment]
> @charlienparker please rebase from clean-bootstrap-4-upgrade, then this can be merged in

done!

**Pull request merged**